packages/yoroi-extension/app/stores/toplevel/WalletSettingsStore.js

```diff
@@ -110,6 +125,71 @@ export default class WalletSettingsStore extends Store<StoresMap, ActionsMap> {
     a.updateSigningPassword.listen(this._changeSigningPassword);
     a.resyncHistory.listen(this._resyncHistory);
     a.removeWallet.listen(this._removeWallet);
+    a.removeTabPermission.listen(this._removeTabPermission)
+    a.requestTabPermission.listen(this._requestTabPermission)
+    a.hidePermissionsDialog.listen(this._hidePermissionsDialog)
+    this._checkConnectorInjectPermission()
+    this._shouldShowPermissionsDialog()
+  }
+
+  @action _shouldShowPermissionsDialog: void => Promise<void> = async () => {
+    // $FlowFixMe
```

Comment:
This is because flow can't figure out the proper typing of user-defined promise type unless it extends `Promise`. So if you want to get rid of the `$FlowFixMe`, I suggest the following:
1. check if `this.getConnectorConfig.result` is non-null, if it is, just use the value
2. otherwise check if it is `isExecuting`, if not, call `.execute`
3. `await this.getConnectorConfig.promise` and assert the value must be non-null